### ISSUE TITLE
Configure Plot

### ISSUE THREAD

**Issue by brombeerberg** [newville/wxmplot#18]
I am new in using wxmplot and find it comfortable. There is one thing I cannot get run: it is the plot configuration. When I choose "right mouse->config" on the plot or "main menu->options->configure plot" nothing happens. It even doesn't work for the simpliest example "WXMPlot Demo". The zoom methods in the context menu work fine.
I guess it has something to do with my OS/installation. I am using Canopy Enthought on Windows 7 and installed wxmplot via pip install.
Grateful for any help.

**Comment by newville** [comment]
@brombeerberg  Sorry for the trouble. I'm not sure what could be going on.  Do I understand correctly that  'right Mouse' does show a pop-up menu,  but that Configure.does not bring up a configuration panel?  

What versions of WxPython and matplotlib do you have?

**Comment by brombeerberg** [comment]
Yes, this is exactly what happens. I have v 2.8.10.1-7 of wxpython and v 1.4.3-6 of matplotlib.

**Comment by newville** [comment]
Oh. 2.8.10 is probably the problem -- that's from 2009.  Is Canopy really that far behind? 

I just checked this worked for on a machine running python 2.6, wxPython 2.8.11, and matploltib 1.1.0, but even 2.8.11 is more than 5 years old.  

Are there any error messages?  I'm not excited about supporting something as old as 2.8.10, but there might be a simple workaround.

**Comment by brombeerberg** [comment]
Okay, I got it running now! You were right, wxpython 2.8.10 was the problem. I have installed  2.8.12 directly into the site-package folder of Canopy. Now the configure dialog appears and works.

It was not possible to update on the "official" way. In the Canopy package manager there is no update available and when entering "pip install --update wxpython" in the Canopy command prompt I was told that wxpython was up to date.

I thank you for your support.
